crimsobot/cogs/cringo.py

```diff
@@ -99,23 +101,21 @@ def join_cringo(reaction: discord.reaction, user: discord.user) -> bool:
 
         # initialize join-message listener
         users_already_joined = []
-        end = time.time() + join_timer
-        while time.time() < end and len(users_already_joined) < 20:
```

Comment:
what's the impetus for removing this timer?

Reply:
The timeout in` wait_for()` is sufficient. I had both the while `time < end` method and the `wait_for(timeout=n)` code in there back when I was test the two methods, but neither appears to be superior to the other. Though I did just lose the player length limiter...